docs/12.EXT-REFERENCE-MODULE.md

```diff
@@ -37,8 +37,7 @@ resolved using the native JerryScript module resolver `jerryx_module_native_reso
 `jerryx_module_resolve()`. Native modules are registered during application startup and by calling `dlopen()` by means
 of library constructors, support for which can be turned on using the `FEATURE_INIT_FINI` build flag. In the absence of
 such a flag, the module registration and unregistration functions are exposed as global symbols which can be called
-explicitly. Note: `FEATURE_INIT_FINI` build flag isn't supported on Windows, because Microsoft Visual C/C++ Compiler
-doesn't support library constructors and destructors.
+explicitly.
```

Comment:
This is not correct, according to this change. FEATURE_INIT_FINI is still not supported on MSVC 2005 and older.